paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -275,54 +275,57 @@ def parse_args() -> argparse.Namespace:
         "--scheduler",
         help="Scheduler to get task info from",
         dest="scheduler",
-        default="mesos",
+        default="kubernetes",
         choices=["mesos", "kubernetes"],
     )
+    parser.add_argument(
+        "--additional-namespaces-exclude",
+        help="full names of namespaces to not fetch allocation info for those that don't match --namespace-prefix-exlude",
+        dest="additional_namespaces_exclude",
+        nargs="+",
+        default=[],
+    )
     parser.add_argument(
         "--namespace-prefix",
-        help="prefix of the namespace to fetch the logs for"
+        help="[UNUSED] prefix of the namespace to fetch the logs for"
         "Used only when scheduler is kubernetes",
         dest="namespace_prefix",
         default="paasta",
     )
     parser.add_argument(
         "--additional-namespaces",
-        help="full names of namespaces to fetch allocation info for that don't match --namespace-prefix"
+        help="[UNUSED] full names of namespaces to fetch allocation info for that don't match --namespace-prefix\n"
         "Used only when scheduler is kubernetes",
         dest="additional_namespaces",
         nargs="+",
         # we default this to tron since this is really the only non-paasta-prefix namespaced that is part of paasta
         # and we'd like to not run two cronjobs to get this information :p
         default=["tron"],
     )
-    return parser.parse_args()
+    args = parser.parse_args()
+
+    args.additional_namespaces_exclude = set(args.additional_namespaces_exclude)
```

Comment:
technically not necessary, but it always makes me twitch when we do an existence check in a list :p